spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/security/saml2/Saml2RelyingPartyRegistrationConfiguration.java

```diff
@@ -28,6 +28,7 @@
 import org.springframework.boot.autoconfigure.security.saml2.Saml2RelyingPartyProperties.Identityprovider.Verification;
 import org.springframework.boot.autoconfigure.security.saml2.Saml2RelyingPartyProperties.Registration;
 import org.springframework.boot.autoconfigure.security.saml2.Saml2RelyingPartyProperties.Registration.Signing;
+import org.springframework.boot.context.properties.PropertyMapper;
```

Comment:
The use of `PropertyMapper` adds a lot of noise to what this PR actually changes. I'd prefer if we did that separately (if at all).

Reply:
If we leave [the associated code](https://github.com/spring-projects/spring-boot/pull/23045/commits/006f58d11e480c5739fa3e0ef229a5f88aea3f57#diff-0639742143d9b9f5e21afa9c54628892L73) as-is, then nulls will override what was specified through the metadata endpoint.

Would surrounding each of those lines with if statements be preferrable?

Reply:
Sorry I missed that. PropertyMapper it is then!